examples/persistent-volume-provisioning/README.md

```diff
@@ -109,6 +109,7 @@ parameters:
   gidMin: "40000"
   gidMax: "50000"
   volumetype: "replicate:3"
+  volumeoptions: "encryption"
```

Comment:
@rootfs my plan is to have a string with more options, for eg, "encryption, performance"..etc in this attribute. So that, internally we will develop a logic in heketi to set volume options based on these values.  So keeping it as meaningful input strings from the user instead of a boolean.